src/analyserequation.cpp

```diff
@@ -16,7 +16,10 @@ limitations under the License.
 
 #include "libcellml/analyserequation.h"
 
+#include <algorithm>
+
 #include "analyserequation_p.h"
+#include "commonutils.h"
```

Comment:
is this needed?

Reply:
Oops, it must have been needed at some point during my work on this PR, but it's clearly not needed anymore. Fixed in #74e5c3c0.